<launch>
	
	<include file="$(find asr_mild_navigation)/launch/simulation/simulation_floor.launch" />
	<include file="$(find asr_next_best_view)/launch/next_best_view_core_sim.launch" />
	<include file="$(find asr_world_model)/launch/world_model.launch" />
	<include file="$(find recognition_for_grasping)/launch/recognition_manager.launch" />
	<include file="$(find asr_flir_ptu_driver)/launch/ptu_left_mock.launch" />
	<node pkg="asr_next_best_view" type="asr_next_best_view_performance_test" name="test" output="screen">
	  <param name="testResultFilePath" value="$(find asr_next_best_view)/testResults"/>
	  <!-- should not contain duplicates, otherwise errors will/may occur -->
	  <rosparam param="nThreads">[1, -1]</rosparam>
	</node>
</launch>
